src/inputs/labeled-inputs.scss

```diff
@@ -229,6 +230,30 @@
     border-top-left-radius: 0;
     border-bottom-left-radius: 0;
   }
+
+  &.iui-actionable {
+    align-items: center;
+    height: 90%;
+    margin-right: 1px;
+    padding: 0 $iui-sm;
+    cursor: pointer;
+    border-radius: 50%;
+
+    @include iui-ripple(var(--iui-color-background-1), var(--iui-color-background-2));
```

Comment:
If I understand correctly it currently always does ripple effect from the center not from the click position. Maybe we could create ripple effect like Material has by adding additional ripple element to do the animation:
![material-ripple](https://user-images.githubusercontent.com/36186912/135570136-de62a7ec-0083-466a-b015-2dd1e3e12cb6.gif)

Reply:
That requires JavaScript, so someday